### PR TITLE
Create section on using quarto

### PR DESCRIPTION
**Pull request by lindsay-fredrick**
Basic section on markdown and quarto. For the advanced pile! Doesn't currently have challenges, might have to think of a few good ones.

### CODE REVIEW

File: course-content/33_create_reports.qmd [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+:::{.callout-tip}
+## All Code Block Execution Options
+
+Here's a list of all the standard execution options we can pass to a codeblock using `#|`:
+
+| Option | Description |
+| ------ | ----------- |
+| `eval` | Evaluate the code chunk (if false, just echos the code into the output). |
+| `echo` | Include the source code in output. |
+| `output` | Include the results of executing the code in the output (true, false, or asis to indicate that the output is raw markdown and should not have any of Quarto’s standard enclosing markdown). |
```
Comment (swayneh) [inline course-content/33_create_reports.qmd]:
```suggestion
| `output` | Include the results of executing the code in the output (true, false, or as is to indicate that the output is raw markdown and should not have any of Quarto’s standard enclosing markdown). |
```

File: course-content/33_create_reports.qmd [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+    <p style="text-align: center">
+    </p>
+    <br/>
+</center>
+
+:::
+
+Notice how the options were applied individually to each format. HTML has a table of contents, the PDF displays no code but all of the outputs, the powerpoint utilized the template we supplied, and the word document uses the defaults! While we have only scratched the surface of what Quarto has to offer, we already see that is a powerful tool that we can use to produce fully reproducible reports, inline documentation, and so much more. 
+
+### Challenge 2
```
Comment (swayneh) [inline course-content/33_create_reports.qmd:470]: Presume you want to add Challenge 2 content

File: course-content/33_create_reports.qmd [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+
+| Right | Left | Default | Center |
+|------:|:-----|---------|:------:|
+|   12  |  12  |    12   |    12  |
+|  123  |  123 |   123   |   123  |
+|    1  |    1 |     1   |     1  |
+
+
+
+### Challenge 1
```
Comment (swayneh) [inline course-content/33_create_reports.qmd:133]: I presume you want to add Challenge 1 content

**Review by swayneh — APPROVED** [review]
Looks really good!  Should be interesting to people I'm sure, whether we get to it in the course or not.

File: course-content/33_create_reports.qmd (hunk 1 above)
Comment (lindsay-fredrick) [reply course-content/33_create_reports.qmd]: Is actually one word - will change to code backticks to be clear.

**Pull request merged**